docker-compose.yaml

```diff
@@ -43,9 +47,14 @@ services:
       - DASHBOARDS_PATH=/etc/grafana/provisioning/dashboards/${DASHBOARD}
       - DISCORD_WEBHOOK_URL=${DISCORD_WEBHOOK_URL}
       - LOKI_URL=http://loki:3100
+      - PROMETHEUS_URL=http://prometheus:9090
       - AWS_AUTH_TYPE=keys
       - AWS_ACCESS_KEY=${AWS_ACCESS_KEY}
       - AWS_SECRET_KEY=${AWS_SECRET_KEY}
       - CLOUDWATCH_METRICS_NAMESPACES=pubsub-dev
     depends_on:
       - promtail
+  ts-agent:
```

Comment:
Should this be renamed to `stats_agent`?

Reply:
yes, done